tests/unit_test/fuel/f3/drivers/driver_manager_test.py

```diff
@@ -37,6 +38,8 @@ def manager(self):
             ("stcp", TcpDriver),
             ("grpc", AioGrpcDriver),
             ("grpcs", AioGrpcDriver),
+            ("nagrpc", GrpcDriver),
+            ("nagrpcs", GrpcDriver),
```

Comment:
minor thing, we have tcp vs atcp,
maybe for grpc we want grpc vs agrpc.
but this does not affect things.